examples/control/laner.py

```diff
@@ -24,7 +24,7 @@ def act(self, obs):
 
 
 def main(scenarios, headless, num_episodes, max_episode_steps=None):
-    agent_specs = {
+    agent_interface = {
```

Comment:
Just revert this for now.
```suggestion
    agent_specs = {
```